assets/pages/weekly-subscription-landing/weeklySubscriptionLandingReducer.js

```diff
@@ -3,44 +3,41 @@
 // ----- Imports ----- //
 
 import type { CommonState } from 'helpers/page/page';
+import { type CountryGroupId } from 'helpers/internationalisation/countryGroup';
 
+import { type WeeklyBillingPeriod, displayPrice } from 'helpers/subscriptions';
 import { type Action } from './weeklySubscriptionLandingActions';
 
 
 // ----- Subs ------ //
-export type SubscriptionWithDetails = {
+export type BillingPeriodWithDetails = {
   title: string,
   offer?: ?string,
-  copy: string,
+  copy: (countryGroupId: CountryGroupId) => string,
 }
 
-type Subscriptions = {
-  weekly: SubscriptionWithDetails,
-  quarterly: SubscriptionWithDetails,
-  annually: SubscriptionWithDetails
-}
-
-export const subscriptions: Subscriptions = {
-  weekly: {
+export const billingPeriods: {
+  [WeeklyBillingPeriod]: BillingPeriodWithDetails
+} = {
+  sixweek: {
     title: 'Weekly',
     offer: 'Introductory offer',
-    copy: '6 issues for 6 pounds and then £37 every 3 months',
+    copy: countryGroupId => `6 issues for 6 pounds and then ${displayPrice('GuardianWeekly', countryGroupId, 'quarter')}`,
```

Comment:
Can we use appropriate currency glyph here e.g. $6, £6 etc instead of saying '6 pounds'?

Reply:
this copy is all temporary!